docs/modules/ROOT/pages/tests-configuration.adoc

```diff
@@ -81,6 +81,10 @@ NOTE: The properties marked with *bold* are mandatory.
 |`statistics.print-failures`
 |`false`
 |If set to `true` prints table of failures including: story, scenario, step and error message
+
+|`report.text-length-diff-threshold`
+|`100`
+|Defines text threshold length to enable unified diff report publishing on failure, for variable comparisons
```

Comment:
I think it makes sense to move this description to the relevant steps to make it transparent for users.

Reply:
We don't have a docs for the step, isn't it out of scope this PR ? :)